docs/conf.py

```diff
@@ -0,0 +1,53 @@
+# Configuration file for the Sphinx documentation builder.
```

Comment:
Remove all the unnecessary comments as in https://github.com/pypa/virtualenv/blob/main/docs/conf.py#L1-L89

Reply:
Will do!